internal/command/postgres/failover.go

```diff
@@ -128,3 +141,170 @@ func runFailover(ctx context.Context) (err error) {
 	fmt.Fprintf(io.Out, "Failover complete\n")
 	return
 }
+
+func flexFailover(ctx context.Context, machines []*api.Machine, app *api.AppCompact) error {
+	io := iostreams.FromContext(ctx)
+
+	leader, err := pickLeader(ctx, machines)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintf(io.Out, "Performing a failover\n")
+
+	primary_region := ""
+	if len(machines) > 0 {
+		for _, machine := range machines {
+			if region, ok := machine.Config.Env["PRIMARY_REGION"]; ok {
+				primary_region = region
+			}
+		}
+	}
```

Comment:
We should verify that all machines agree on what the PRIMARY_REGION is.  A failed deploy or manual machine update could leave things in an inconsistent state.